met-web/src/components/engagement/admin/create/authoring/AuthoringTemplate.tsx

```diff
@@ -0,0 +1,192 @@
+                        In the Details Section of your engagement, you have the option to display your content in a
+                        normal, static page section view (No Tabs), or for lengthy content, use Tabs. You may wish to
+                        use tabs if your content is quite lengthy so you can organize it into smaller, more digestible
+                        chunks and reduce the length of your engagement page.
+                    </EyebrowText>
+                    <RadioGroup
+                        row
+                        id="tabs_enabled"
+                        name="tabs_enabled"
+                        defaultValue="false"
+                        value={contentTabsEnabled}
+                        onChange={handleTabsRadio}
+                        sx={{ flexWrap: 'nowrap', fontSize: '0.8rem', mb: '1rem', width: '100%' }}
+                    >
+                        <Grid item xs={6}>
+                            <FormControlLabel value="false" control={<Radio />} label="No Tabs" />
+                        </Grid>
+                        <Grid item xs={6}>
+                            <FormControlLabel value="true" control={<Radio />} label="Tabs (2 Minimum)" />
+                        </Grid>
+                    </RadioGroup>
+                </Grid>
+            )}
+            <Header2 decorated style={{ paddingTop: '1rem' }}>
+                <Suspense>
+                    <Await resolve={languages}>
+                        {(languages: Language[]) => getLanguageValue(currentLanguage, languages) + ' Content'}
+                    </Await>
+                </Suspense>
+            </Header2>
+
+            <Form onSubmit={handleSubmit(onSubmit)}>
+                <Suspense>
+                    <Await resolve={engagement}>
+                        {(engagement: Engagement) => (
+                            <Outlet
+                                context={{
+                                    setValue,
+                                    watch,
+                                    setTabs,
+                                    setSingleContentValues,
+                                    setContentTabsEnabled,
+                                    control,
+                                    engagement,
+                                    contentTabsEnabled,
+                                    tabs,
+                                    singleContentValues,
+                                    defaultTabValues,
+                                }}
+                            />
+                        )}
+                    </Await>
+                </Suspense>
+
+                <UnsavedWorkConfirmation blockNavigationWhen={isDirty && !isSubmitting} />
+                <Suspense>
+                    <Await resolve={languages}>
+                        {(languages: Language[]) => (
+                            <AuthoringBottomNav
+                                isDirty={isDirty}
```

Comment:
Nice clean code using form context to pass isDirty into the bottom nav ^_^

Reply:
thanks!